patches/aspnetcore/0001-Use-non-portable-NETCoreAppRuntime-for-crossgen.patch

```diff
@@ -0,0 +1,50 @@
+From 77b1e19ae4f729b9c8dc9e4987dd4f0ccbb2e73d Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Wed, 13 May 2020 19:46:47 +0000
+Subject: [PATCH 01/13] Use non-portable NETCoreAppRuntime for crossgen
+
+---
+ eng/Dependencies.props                                      | 1 +
+ .../App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj | 6 ++++--
+ 2 files changed, 5 insertions(+), 2 deletions(-)
+
+diff --git a/eng/Dependencies.props b/eng/Dependencies.props
+index 8fc6f8364f..8107003c09 100644
+--- a/eng/Dependencies.props
++++ b/eng/Dependencies.props
+@@ -93,6 +93,7 @@ and are generated based on the last package release.
+     <LatestPackageReference Include="Microsoft.NETCore.App.Runtime.linux-arm64" />
+     <LatestPackageReference Include="Microsoft.NETCore.App.Runtime.linux-musl-x64" />
+     <LatestPackageReference Include="Microsoft.NETCore.App.Runtime.linux-musl-arm64" />
++    <LatestPackageReference Condition="'$(TargetOsName)' != 'osx'" Include="microsoft.netcore.app.runtime.$(SourceBuildRuntimeIdentifier)" Version="$(MicrosoftNETCoreAppRuntimeVersion)" />
```

Comment:
Package name case is different, and custom Version seems unnecessary, should be assigned below:

```xml
    <LatestPackageReference Update="@(LatestPackageReference->WithMetadataValue('Version', ''))">
      <Version
          Condition=" $([System.String]::new('%(Identity)').StartsWith('Microsoft.NETCore.App.Runtime.')) ">$(MicrosoftNETCoreAppRuntimeVersion)</Version>
```

Probably won't hurt, but would be nice to have this mergeable. On that note, could also slap a `'$(DotNetBuildFromSource)' == 'true'` on it.